owsi-core/owsi-core-components/owsi-core-component-jpa-migration/src/main/java/fr/openwide/core/jpa/migration/service/AbstractBatchAssociationMigrationService.java

```diff
@@ -31,30 +32,21 @@
 		extends AbstractMigrationService {
 	
 	private static final int DEFAULT_VALUES_PER_KEY = 3;
+	
+	@Autowired
+	private BatchCreator batchCreator;
```

Comment:
Probably. I preferred a BatchCreator class so that the different possibilities are autodocumented.